recipes/vtk/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+mkdir build
+cd build
+
+BUILD_CONFIG=Release
+
+# sometimes python is suffixed, this is a quick fix
+# in a future PR we should probably switch to cmake find python scripting
+PYTHON_INCLUDE=${PREFIX}/include/python${PY_VER}
+if [ ! -d $PYTHON_INCLUDE ]; then
+  PYTHON_INCLUDE=${PREFIX}/include/python${PY_VER}m
+fi
+
+PYTHON_LIBRARY="libpython${PY_VER}.so"
+PYTHON_LIBRARY=${PREFIX}/lib/${PYTHON_LIBRARY}
+if [ ! -f $PYTHON_LIBRARY ]; then
+    PYTHON_LIBRARY=${PREFIX}/lib/libpython${PY_VER}m.so
+fi
+
+cmake .. -G "Unix Makefiles" \
+    -Wno-dev \
+    -DCMAKE_BUILD_TYPE=$BUILD_CONFIG \
+    -DCMAKE_INSTALL_PREFIX:PATH="${PREFIX}" \
+    -DCMAKE_INSTALL_RPATH:PATH="${PREFIX}/lib" \
+    -DINSTALL_PKGCONFIG_DIR:PATH=$PKG_CONFIG_PATH \
```

Comment:
`INSTALL_PKGCONFIG_DIR` is unused according to a CMake warning. will remove that in future PR.